cli/tests/type_headers_deno_types.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Following import uses two distinct ways to provide types:
+ * - X-TypeScript-Types headers
+ * - @deno-types directive
+ * 
+ * Because "@deno-types" directive must be placed by user explicitly it
+ * should have higher precedence than type header.
```

Comment:
Here you write that `@deno-types` should have higher precedence, but in the title you say `X-TypeScript-Types` should have higher precedence. Which is it?

Reply:
`@deno-types` should have higher precedence